core/src/main/java/io/aiven/klaw/service/TopicControllerService.java

```diff
@@ -1067,9 +1067,12 @@ public int compare(Topic topic1, Topic topic2) {
     }
   }
 
-  static class TopicNameSyncComparator implements Comparator<TopicConfig> {
+  static class TopicNameSyncComparator
+      implements Comparator<io.aiven.klaw.model.response.TopicConfig> {
     @Override
-    public int compare(TopicConfig topic1, TopicConfig topic2) {
+    public int compare(
+        io.aiven.klaw.model.response.TopicConfig topic1,
```

Comment:
do we need full namespace of TopicConfig here ?

Reply:
No we dont and I saw this earlier and forgot to get rid of it :/